io.gpl.wxmaxima.yml

```diff
@@ -0,0 +1,71 @@
+app-id: io.gpl.wxmaxima
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+modules:
+  - name: gnuplot
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app/ --with-qt=no
+      - make
+      - make install
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/gnuplot-5.4.10.tar.gz
+        sha256: 975d8c1cc2c41c7cedc4e323aff035d977feb9a97f0296dd2a8a66d197a5b27c
+  - name: maxima
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - cp -r ./sbcl/* /app/
+      - mkdir bbbb
+      - cd bbbb && ../configure --enable-sbcl --prefix=/app/
+      - cd bbbb && make
+      - cp doc/info/build-html-index.lisp bbbb/doc/info/
+      - cd bbbb && make install 
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/maxima-5.47.0.tar.gz
+        sha256: 9104021b24fd53e8c03a983509cb42e937a925e8c0c85c335d7709a14fd40f7a
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/sbcl-2.3.9.tar.gz
+        sha256: 4b685a3b834d1faf0976e936750062e4c8ad5718daf115075615402913b69121
+  - name: wxwidgets
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - mkdir bb
+      - cd bb && ../configure --with-gtk --enable-webviewwebkit --prefix=/app/
+      - cd bb && make
+      - cd bb && make install
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/wxWidgets-3.2.2.1.tar.bz2
+        sha256: dffcb6be71296fff4b7f8840eb1b510178f57aa2eb236b20da41182009242c02
+  - name: wxmaxima
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - mkdir build-wxm
+      - cd build-wxm && cmake -DCMAKE_INSTALL_PREFIX=/app/ -DwxWidgets_INCLUDE_DIRS=/app/include/wx-3.2 -DwxWidgets_LIBRARIES=app/lib -DWXM_DISABLE_WEBVIEW=on ../
+      - cd build-wxm && cmake --build .
+      - cd build-wxm && cmake --build . -- install
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/wxmaxima-Version-23.10.0.tar.gz
+        sha256: 1c41352be0d88dc3b2413bab06f0a2e791579d61ac1e3a406c3349f5ea0fffe7
+  - name: desktop
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - install -Dm0644 wxMaxima128.png /app/share/icons/hicolor/128x128/apps/io.gpl.wxmaxima.png
+      - install -Dm0644 io.gpl.wxmaxima.desktop /app/share/applications/io.gpl.wxmaxima.desktop
```

Comment:
I'll guess that you are also probably missing the AppStream file.

Reply:
It looks like that upstream contains AppStream metadata file: https://github.com/wxMaxima-developers/wxmaxima/blob/main/data/io.github.wxmaxima_developers.wxMaxima.appdata.xml

It also contains desktop and icon files: https://github.com/wxMaxima-developers/wxmaxima/tree/main/data

Reply:
yes, see https://github.com/flathub/flathub/pull/4641#discussion_r1375337636